frontend/src/lib/constants.tsx

```diff
@@ -170,6 +170,28 @@ export const FEATURE_MINIMUM_PLAN: Record<AvailableFeature, LicensePlan> = {
     [AvailableFeature.ROLE_BASED_ACCESS]: LicensePlan.Enterprise,
     [AvailableFeature.RECORDINGS_FILE_EXPORT]: LicensePlan.Scale,
     [AvailableFeature.RECORDINGS_PERFORMANCE]: LicensePlan.Scale,
+    [AvailableFeature.EVENTS]: LicensePlan.Starter,
```

Comment:
Not sure if we need to implement this now, but maybe on a subsequent PR where we use the `available_features` list from the billing service to check for feature access. I'm worried this could break existing paygates.